v2.0/enable-node-map.md

```diff
@@ -1,7 +1,200 @@
 ---
-title: Enable the Node Map
+title: Enable the Node Map (Enterprise)
 summary: Learn how to enable the node map in the Admin UI.
 toc: false
 ---
 
-*Docs coming soon*
\ No newline at end of file
+The **Node Map** visualizes the geographical configuration of a multi-regional cluster by plotting the node localities on a world map. The **Node Map** also provides real-time cluster metrics, with the ability to drill down to individual nodes to monitor and troubleshoot the cluster health and performance.
```

Comment:
Add the "new in v2.0" callout to the beginning of this paragraph. Please do that same on the Table Partitioning page.

Reply:
Added to this doc. Made note to add it to the partitioning PR.